src/jvm/test/org/jetbrains/kotlinx/lincheck_test/gpmc/PrintingTest.kt

```diff
@@ -19,7 +19,7 @@ import kotlin.concurrent.thread
  * This test aims to test the effects of printing to `stdout` and `stderr`.
  * The test used to hang because of prints, see https://github.com/JetBrains/lincheck/issues/409.
  *
- * See [CyclicBarrierTest] for identical test but with no printing.
+ * See `CyclicBarrierTest` for identical test but with no printing.
  */
 class PrintingTest {
```

Comment:
If you moved the `CyclicBarrierTest` to integration tests, would it make sense to also move this one? 


Or, in general, are there any other gpmc tests worth moving into integration tests?
I am trying to understand our long-term strategy here: what tests should we move to integration tests? The moving itself we can perform later in separate PR.